src/components/TeamLocations/TeamLocations.jsx

```diff
@@ -321,49 +316,49 @@ function TeamLocations() {
           </div>
         ) : null}
       </div>
-      <div style={{position: 'relative'}}>
-      <div>{tableVisible && <TeamLocationsTable visible={tableVisible} mapMarkers={mapMarkers} setCurrentUser={setCurrentUser} darkMode={darkMode} />}</div>
-      <MapContainer
-        id='map-container'
-        center={[51.505, -0.09]}
-        maxBounds={[
-          [-90, -225],
-          [90, 225],
-        ]}
-        maxBoundsViscosity={1.0}
-        zoom={3}
-        scrollWheelZoom
-        style={{ border: '1px solid grey' }}
-        ref={mapRef}
-      >
-        <EventComponent setPopupsOpen={setPopupsOpen} currentUser={currentUser} setMarkerPopupVisible={setMarkerPopupVisible}  />
-        
-          
-           
-           
-          
-          <TileLayer
-          attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
-          url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-          minZoom={2}
-          maxZoom={15} />
-         
-        <MarkerClusterGroup disableClusteringAtZoom={13} spiderfyOnMaxZoom={true} chunkedLoading>
-          {tableVisible && currentUser ?  
-          
-          <MarkerPopup
-            key={currentUser._id}
-            profile={currentUser}
-            userName={getUserName(currentUser)}
-            isAbleToEdit={isAbleToEdit}
-            editHandler={editHandler}
-            removeLocation={removeLocation}
-            isOpen={markerPopupVisible}
-            darkMode={darkMode} /> 
-            
-            : markerPopups }
-        </MarkerClusterGroup>
-      </MapContainer>
+      <div style={{ position: 'relative' }}>
+        <div>{tableVisible && <TeamLocationsTable visible={tableVisible} filteredMapMarkers={filteredMapMarkers} setCurrentUser={setCurrentUser} darkMode={darkMode} />}</div>
```

Comment:
This line is the reason of the table view icon console error.

Reply:
The error line is correct, but the error happened in TeamLocationsTable component due to change in field name from mapMarkers to filteredMapMarkers. I have fixed it, also the dark theme was missing in the table view which i have added. Pls review the changes!

Reply:
Thanks for fixing the error, it's working now after changes in [63beca7](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2515/commits/63beca7efa8b1c1a92ce950d7e5e6656ab4ead2c)